CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+This document explains the developer workflow for making changes to the
+[Metals extension](https://marketplace.visualstudio.com/items?itemName=scalameta.metals)
```

Comment:
This should be `[Scala Syntax](https://marketplace.visualstudio.com/items?itemName=scala-lang.scala)`

Reply:
Nice catch, the release script also referenced metals  which is fixed now.